draft-ietf-teep-architecture.md

```diff
@@ -1134,21 +1123,22 @@ for protecting the resource usage allocated for TA management.
 
 ## Compromised CA
 
-A root CA for TAM certificates might get compromised. A Trust Anchor other 
-than a root CA certificate may also be compromised. Some TEE Trust
+TAMs are responsible for validating the TEE certificate chain, including
+the TEE certificate and any intermediate CA certificates up to the root
+certificate. Such validation includes checking for certificate revocation.
+A root CA for TAM certificates might get compromised or its certificate might
+expire. A Trust Anchor other 
```

Comment:
The paragraph isn't consistent. Do you mean "A root CA for TEE certificates might get compromised..."? The section talks about TAM validates TEE certificates, and it should continue to say the root about TEE certs, not switch to a root CA for TAM certificates.

Reply:
Thanks for catching this!  Rearranged text, please recheck